README.md

```diff
@@ -94,7 +94,11 @@ Note that if you get errors, run the above commands with `--verbose` - this will
 
```

Comment:
Should the device config on ln 46 onwards change to reflect the new transport schema? (host and key rather than ip and key)?

Reply:
Updated